pulsar-client/src/main/java/org/apache/pulsar/client/impl/MultiplierRedeliveryBackoff.java

```diff
@@ -0,0 +1,95 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.client.impl;
+
+import com.google.common.base.Preconditions;
+import org.apache.pulsar.client.api.RedeliveryBackoff;
+
+/**
+ * MultiplierRedeliveryBackoff
+ */
+public class MultiplierRedeliveryBackoff implements RedeliveryBackoff {
+
+    private final long minDelayMs;
+    private final long maxDelayMs;
+    private final int multiplier;
+    private final int maxMultiplierPow;
+
+    private MultiplierRedeliveryBackoff(long minDelayMs, long maxDelayMs, int multiplier) {
```

Comment:
Better to add some check here, maxDelayMs should >= minDelayMs, and multiplier should > 0?

Reply:
https://github.com/apache/pulsar/blob/2e4a98138ede678cc3d06a885de6a60dad796c99/pulsar-client/src/main/java/org/apache/pulsar/client/impl/MultiplierRedeliveryBackoff.java#L90  has checked